.github/workflows/release.yml

```
@@ -39,61 +39,23 @@ jobs:
           files: |
             dist/artifacts/kine-*
             dist/artifacts/*.txt
-
-      - name: "Read Vault secrets"
-        if: github.repository_owner == 'k3s-io'
-        uses: rancher-eio/read-vault-secrets@main
-        with:
-          secrets: |
-            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
-            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_TOKEN ;
-    
-      - name: Login to DockerHub with Rancher Secrets
-        if: github.repository_owner == 'k3s-io'
-        uses: docker/login-action@v3
-        with:
-          username: ${{ env.DOCKER_USERNAME }}
-          password: ${{ env.DOCKER_TOKEN }}
-
-      # For forks, setup DockerHub login with GHA secrets
-      - name: Login to DockerHub with GHA Secrets
-        if: github.repository_owner != 'k3s-io'
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_TOKEN }}
       
-      - name: Login to GitHub Container Registry
+      - name: Login to GitHub Container Registry (loft-sh)
+        if: github.repository_owner == 'loft-sh'
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
-          username: ${{ github.repository_owner }}
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Let's use `github.actor` for username and `GHCR_TOKEN` for password when pushing to ghcr.io.
```suggestion
          username: ${{ github.actor }}
          password: ${{ secrets.GHCR_TOKEN }}
```